libs/model/src/test/seedDb.ts

```diff
@@ -367,36 +344,34 @@ export const seedDb = async (debug = false): Promise<void> => {
         description: 'A topic made for testing',
         community_id: 'ethereum',
       },
-    ];
-    await models.Topic.bulkCreate(topics);
+    ]);
 
-    const contracts: ContractAttributes[] = [
-      {
-        address: '0xFab46E002BbF0b4509813474841E0716E6730136',
-        token_name: 'Alex',
-        symbol: 'ALEX',
-        type: ChainNetwork.ERC20,
-        chain_node_id: testnetNode.toJSON().id!,
-      },
-      {
-        address: '0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e',
-        token_name: 'yearn',
-        symbol: 'YFI',
-        type: ChainNetwork.ERC20,
-        chain_node_id: mainnetNode.toJSON().id!,
-      },
-      {
-        address: '0x6b3595068778dd592e39a122f4f5a5cf09c90fe2',
-        token_name: 'sushi',
-        symbol: 'SUSHI',
-        type: ChainNetwork.ERC20,
-        chain_node_id: mainnetNode.toJSON().id!,
-      },
-    ];
     const [alexContract, yearnContract, sushiContract] =
-      await models.Contract.bulkCreate(contracts);
+      await models.Contract.bulkCreate([
+        {
+          address: '0xFab46E002BbF0b4509813474841E0716E6730136',
+          token_name: 'Alex',
+          symbol: 'ALEX',
+          type: ChainNetwork.ERC20,
+          chain_node_id: testnetNode.toJSON().id!,
+        },
+        {
+          address: '0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e',
+          token_name: 'yearn',
+          symbol: 'YFI',
+          type: ChainNetwork.ERC20,
+          chain_node_id: mainnetNode.toJSON().id!,
+        },
+        {
+          address: '0x6b3595068778dd592e39a122f4f5a5cf09c90fe2',
+          token_name: 'sushi',
+          symbol: 'SUSHI',
+          type: ChainNetwork.ERC20,
+          chain_node_id: mainnetNode.toJSON().id!,
+        },
+      ]);
 
-    const communityContracts: CommunityContractAttributes[] = [
+    await models.CommunityContract.bulkCreate([
       {
         community_id: alex.toJSON().id!,
```

Comment:
Why are we using `toJSON()` in these assignments? `id` should be accessible directly on the returned sequelize object object and converting to JSON seems like unnecessary processing.